src/Mod/TechDraw/Gui/TaskBalloon.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>307</width>
-    <height>279</height>
+    <height>328</height>
```

Comment:
Why do you need 48 px more space? I tried to keep the dialogs as small as necessary because on my small 15 inch laptop screen there were often too big.